dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -606,13 +608,13 @@ public WorkflowScheme findSystemWorkflowScheme() throws DotDataException {
 
 	@Override
 	@CloseDBIfOpened
-	public WorkflowScheme findScheme(final String id) throws DotDataException, DotSecurityException {
+	public WorkflowScheme findScheme(final String idOrVar) throws DotDataException, DotSecurityException {
```

Comment:
shouldn't we have a findScheme test by variable in the WorkflowAPITest?

Reply:
Done